cmd/operator-sdk/bundle/validate.go

```diff
@@ -105,24 +121,56 @@ func (c bundleValidateCmd) validate(args []string) error {
 func (c *bundleValidateCmd) addToFlagSet(fs *pflag.FlagSet) {
 	fs.StringVarP(&c.imageBuilder, "image-builder", "b", "docker",
 		"Tool to extract container images. One of: [docker, podman]")
+	fs.StringVarP(&c.outputFormat, "output", "o", "text",
+		"Output format for results.  Valid values: text, json-alpha1 (Note that it is in alpha stage)")
```

Comment:
I think the `-alpha1` suffix makes the value obviously in alpha right?
```suggestion
		"Output format for results.  Valid values: text, json-alpha1")
```